### PR TITLE
fix: add missing fields in organizer event emails

### LINKED ISSUE #8261: Organizer Event Emails for Attendee Orders are Missing Postcode, City, State and Country and change to "Purchase on Invoice"
Please add Postcode, City, State and Country to the organizer event emails for attendee orders, if they have been filled into the billing information.

For payment methods change "invoice" to "Purchase on invoice".

Ensure that the first letter is a capital after a colon.

Screenshot from 2021-10-04 19-43-06

### LINKED ISSUE #8263: Attendee Ticket Emails: Add Date and Time to Subject
Please change the subjects of attendee tickets emails from "See you at [Conference Name] ([Order Number])" to "See you at [Conference Name] on [Event Date] at [Event Time] [Timezone] ([Order Number])"

Screenshot from 2021-10-04 19-45-22

### PR DESCRIPTION
**Pull request by progmatic-99**
Fixes #8261 
Also Fixes #8263 

ss_1633434641

**Attendee email**
ss_1633499422

### CODE REVIEW

**Comment by maze-runnar** [comment]
@progmatic-99 can you please fix https://github.com/fossasia/open-event-server/issues/8263 in same PR

File: app/api/helpers/mail.py [hunk 1]
```diff
@@ -543,6 +546,13 @@ def send_email_to_attendees(order):
         order_view_url=order.site_view_link,
     )
 
+    event_date = event.starts_at.strftime('%d %B %Y')
+    event_time = (
+        event.starts_at.replace(tzinfo=pytz.timezone('UTC'))
+        .astimezone(pytz.timezone(event.timezone))
+        .strftime('%H:%M (%Z%z)')
+    )
+
```
Comment (maze-runnar) [inline app/api/helpers/mail.py:555]: at one place you have used `.replace(tzinfo=pytz.timezone('UTC'))
        .astimezone(pytz.timezone(event.timezone))` and other not. Any specific reason?

Comment (progmatic-99) [reply app/api/helpers/mail.py:555]: Which other place you are talking about??

If you are asking about why I have changed only 1 attendee email, @mariobehling asked me to do so.

Comment (mariobehling) [reply app/api/helpers/mail.py:555]: Well, I think the event time and time zone should be handled in the same way in both emails. But, lets try to get together in the meeting so we are all on the same page. @progmatic-99 Could you open another issue for the second email please. So, we can finalize this PR? Thanks.

Comment (progmatic-99) [reply app/api/helpers/mail.py:555]: Ok @mariobehling

**Comment by maze-runnar** [comment]
@progmatic-99 please complete this PR, as we discussed in the meeting.

File: app/api/helpers/mail.py [hunk 2]
```diff
@@ -730,3 +745,13 @@ def send_email_to_moderator(video_stream_moderator):
             settings=get_settings(),
         ),
     )
+
+
+def convert_to_event_timezone(date, timezone=None, format='%B %d, %Y %H:%M (%Z%z)'):
+    if not date:
+        return ''
```
Comment (maze-runnar) [inline app/api/helpers/mail.py:752]: Return None

**Pull request merged**